packages/core/admin/admin/src/pages/UsecasePage/index.js

```diff
@@ -0,0 +1,148 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { useIntl } from 'react-intl';
+import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+import { pxToRem, useNotification } from '@strapi/helper-plugin';
+import { Main } from '@strapi/design-system/Main';
+import { Flex } from '@strapi/design-system/Flex';
+import { Box } from '@strapi/design-system/Box';
+import { Stack } from '@strapi/design-system/Stack';
+import { Typography } from '@strapi/design-system/Typography';
+import { Select, Option } from '@strapi/design-system/Select';
+import { TextInput } from '@strapi/design-system/TextInput';
+import { TextButton } from '@strapi/design-system/TextButton';
+import { Button } from '@strapi/design-system/Button';
+import Logo from '../../components/UnauthenticatedLogo';
+import UnauthenticatedLayout, { LayoutContent } from '../../layouts/UnauthenticatedLayout';
+
+export const options = [
+  { label: 'Front-end developer', value: 'front_end_developer' },
+  { label: 'Back-end developer', value: 'back_end_developer' },
+  { label: 'Full-stack developer', value: 'full_stack_developer' },
+  { label: 'Content Manager', value: 'content_manager' },
+  { label: 'Content Creator', value: 'content_creator' },
+  { label: 'Other', value: 'other' },
+];
+
+const TypographyCenter = styled(Typography)`
+  text-align: center;
+`;
+
+const UsecasePage = () => {
+  const toggleNotification = useNotification();
+  const { push, location } = useHistory();
+  const { formatMessage } = useIntl();
+  const [persona, setPersona] = useState();
+  const [personaOther, setPersonaOther] = useState('');
+
+  const isOther = persona === 'other';
+  const isComingFromRegister = location.state?.fromRegister;
+
+  useEffect(() => {
+    if (!isComingFromRegister) {
+      push('/');
+    }
+  }, [isComingFromRegister, push]);
+
+  const handleSubmit = skipPersona => {
+    try {
+      const {
+        state: { email, firstAdmin, firstname },
+      } = location;
+
+      axios({
+        method: 'POST',
+        url: 'https://analytics.strapi.io/register',
+        data: {
+          email,
+          username: firstname,
```

Comment:
That's seems odd to me, so just to confirm: sure you want to send firstname through username? :)

Reply:
😅 yes it is confusing - it's a code legacy from the tracker side!
before the username was required to create an account but since it changed we have this legacy. 
I talked with @fdel-car about this and we'll need to fix it at some point but not for this feature!

Reply:
We were all very surprised by this (and you're lucky to have discovered it from strapi's side and not the tracker 😂).

But yes, Alex is open to the idea of updating the tracker and dropping events (allowing us to update this code) but we'll need to be careful.

Reply:
If you could do on another PR that would be awesome

Reply:
I'm not sure who will handle this tracker update, since it's sensible we might need to discuss it before and add it to a sprint. I think @fdel-car is already working for a tracker v2, I don't know if it's in their backlog. WDYT @soupette 
For the Usecase Persona feature though we need to move on with it if possible, it was due for this sprint!